coursedata/quizzes/en.yaml

```diff
@@ -495,283 +497,284 @@ levels:
             code: |-
-                print 'Welcome at the money show !'
-                print 'in front of you are 3 doors'
-                door is ask 'Which door do you choose ?'
+                {print} 'Welcome at the money show!'
+                {print} 'in front of you are 3 doors'
+                door is {ask} 'Which door do you choose?'
             mp_choice_options:
-                -   "option_text": "print So you pick door door"
+                -   "option_text": "{print} So you pick door door"
                     "feedback": "We need quotation marks"
-                -   "option_text": "print 'So you pick ' door door"
+                -   "option_text": "{print} 'So you pick ' door door"
                     "feedback": "If the player chooses door 3, Hedy will say 'So you pick 3 3"
-                -   "option_text": "print 'So you pick door ' door"
-                    "feedback": "Super !"
-                -   "option_text": "print 'So you pick door door'"
-                    "feedback": "Hedy will literally print 'So you pick door door"
+                -   "option_text": "{print} 'So you pick door ' door"
+                    "feedback": "Super!"
+                -   "option_text": "{print} 'So you pick door door'"
+                    "feedback": "Hedy will literally {print} 'So you pick door door"
             correct_answer: "C"
             hint: "The second word door should be replaced with the number, the first should\
             \ still be the word door..."
             question_score: '10'
         9:
-            question_text: "What will never appear in your output screen ?"
+            question_text: "What will never appear in your output screen?"
             code: |-
                 clubs is Real Madrid, Bayern Munchen, Manchester United, Ajax
-                print clubs at random ' is going the win the champions league'
+                {print} clubs {at} {random} ' is going the win the champions league'
             mp_choice_options:
                 -   "option_text": "Ajax is going to win the champions league"
-                    "feedback": "Hedy could print that"
+                    "feedback": "Hedy could {print} that"
                 -   "option_text": "Real Madrid is going to win the champions league"
-                    "feedback": "Hedy could print that"
+                    "feedback": "Hedy could {print} that"
                 -   "option_text": "Bayer Munchen is going to win the champions league"
-                    "feedback": "Hedy could print that"
+                    "feedback": "Hedy could {print} that"
                 -   "option_text": "FC Barcelona is going to win the champions league"
                     "feedback": "That's right. It's not in the list"
             correct_answer: "D"
-            hint: "What are Hedy's options to randomly pick from ?"
+            hint: "What are Hedy's options to {random}ly pick from?"
```

Comment:
```suggestion
            hint: "What are Hedy's options to randomly pick from?"
```

Reply:
Can I just press 'Commit suggestion' on all these buttons here? I haven't seen this before :)